src/Analyzer/Resolve/IdentifierResolver.cpp

```diff
@@ -417,11 +417,7 @@ QueryTreeNodePtr IdentifierResolver::tryResolveTableIdentifierFromDatabaseCatalo
     bool is_temporary_table = storage_id.getDatabaseName() == DatabaseCatalog::TEMPORARY_DATABASE;
 
     StoragePtr storage;
-
-    if (is_temporary_table)
-        storage = DatabaseCatalog::instance().getTable(storage_id, context);
-    else
-        storage = DatabaseCatalog::instance().tryGetTable(storage_id, context);
+    storage = DatabaseCatalog::instance().getTable(storage_id, context);
```

Comment:
What's the logic behind this change?

Reply:
I want it to throw an error in CI environment for debugging purposes. I will revert the change later.